.gitignore

```diff
@@ -1,4 +1,5 @@
 .vscode/
+.idea/
```

Comment:
you should put this in your global .gitconfig instead.

Reply:
gotcha